.github/workflows/ci.yml

```diff
@@ -87,10 +87,10 @@ jobs:
         # Depth of at least 2 for codecov coverage diffs. See https://github.com/GoogleChrome/lighthouse/pull/12079
         fetch-depth: 2
 
-    - name: Use Node.js 12.x
```

Comment:
just updating to demonstrate that everything works as expected in the current node LTS (14). Otherwise, there's a few `lookupLocale` tests that fail in node 12 because they all return `'en'`.

If we maintain Node 12 support, we could e.g. unit test on Node 12 and node 14, and assert either the correct locale is returned or we're in a `small-icu` build of node and a warning was logged.

Reply:
ok?